engine/src/test/java/com/redhat/cloud/notifications/processors/camel/CamelTypeProcessorTest.java

```diff
@@ -98,7 +100,9 @@ void testCamelEndpointProcessing() {
         // Two endpoints should have been processed.
         assertEquals(2, result.size());
         // Metrics should report the same thing.
-        micrometerAssertionHelper.assertCounterIncrement(PROCESSED_COUNTER_NAME, 2);
+        micrometerAssertionHelper.assertCounterIncrement(PROCESSED_COUNTER_NAME, 2, SUB_TYPE_KEY, SUB_TYPE);
```

Comment:
The behavior of MicrometerAssertionHelper will be unreliable if `saveCounterValuesBeforeTest` isn't called before the tests (see `beforeEach()`) with the new `counter name + tags` combination.